README.md

```diff
@@ -1,32 +1,41 @@
 # nz.co.fuzion.civixero
 
-Synchronisation between CiviCRM &amp; Xero
+Synchronisation [CiviCRM](https://civicrm.org) and [Xero](https://xero.com) for financial transactions and contacts.
 
-This extension requires the extension https://github.com/eileenmcnaughton/nz.co.fuzion.accountsync to work.
+This extension requires the [AccountSync extension](https://github.com/eileenmcnaughton/nz.co.fuzion.accountsync).
 
 It sets up scheduled jobs that synchronize Xero contacts and invoices with CiviCRM contacts and invoices.
 
 Interaction with this module is primarily by API and it creates scheduled jobs to run those API.
 
-## Customisation / Additional Extensions
+## Customisation &amp; Additional Extensions
 
-To modify the behaviour of the civixero extension various hooks are available.
+To modify the behaviour of the CiviXero extension various hooks are available. The additional CiviCRM extensions are available:
 
-See https://github.com/agileware/au.com.agileware.xerotweaks for an example usage of this.
+[Xero Tweaks](https://github.com/agileware/au.com.agileware.xerotweaks)
+- Removes the Contact ID from their Xero name.
+- Includes additional address lines in the contact address.
+- Removes the Contact's name from the Invoice Reference and Line items.
+
+[Xero Untax](https://github.com/agileware/au.com.agileware.xerountax)
+- Remove tax details from line items sent via CiviXero, so Xero can figure it out avoiding rounding issues.
```

Comment:
Dang - I didn't think we were passing through tax details - did someone convince me to?

Reply:
That would have been our PR #18

Reply:
ah...